code/modules/mob/living/carbon/human/species_types/shadowpeople.dm

```diff
@@ -68,6 +67,72 @@
 			return -1
 	return 0
 
+
+
+//Organ
+
+/obj/item/organ/heart/nightmare
+	name = "heart of darkness"
+	desc = "An alien organ that twists and writhes when exposed to light."
+	icon = 'icons/obj/surgery.dmi'
+	icon_state = "demon_heart-on"
+	color = "#1C1C1C"
+	var/respawn_progress = 0
+	var/obj/item/light_eater/blade
+
+
+
+/obj/item/organ/heart/nightmare/attack(mob/M, mob/living/carbon/user, obj/target)
+	if(M != user)
+		return ..()
+	user.visible_message("<span class='warning'>[user] raises [src] to their mouth and tears into it with their teeth!</span>", \
+						 "<span class='danger'>[src] feels unnaturally cold in your hands. You raise [src] your mouth and devour it!</span>")
+	playsound(user, 'sound/magic/demon_consume.ogg', 50, 1)
+
+
+	user.visible_message("<span class='warning'>Blood erupts from [user]'s arm as it reforms into a weapon!</span>", \
+						 "<span class='userdanger'>Icy blood pumps through your veins as your arm reforms itself!</span>")
+	user.drop_item()

```

Comment:
user.temporarilyRemoveItemFromInventory(src, TRUE)

Reply:
That'a neat